demisto_sdk/commands/common/tests/pack_unique_files_test.py

```diff
@@ -208,3 +208,24 @@ def test_is_approved_tags(self, repo, tags, is_valid):
         assert self.validator._is_approved_tags() == is_valid
         if not is_valid:
             assert 'The pack metadata contains non approved tags: NonApprovedTag' in self.validator.get_errors()
+
+    @pytest.mark.parametrize('type, is_valid', [
+        ('community', True),
+        ('partner', True),
+        ('xsoar', True),
```

Comment:
please also add test for `developer`

Reply:
@ShahafBenYakir isn't the pack support of type developer deprecated?